src/components/todo/TodoListItem.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import './TodoListItem.css';
+
+const TodoListItem = ({todo, onRemovePress, onCompletedPress}, key) => {
+    return (
+        <div className="todo-item-container" key={key}>
```

Comment:
what is this key for? I don't think you need it

Reply:
> what is this key for? I don't think you need it

I get console error if I don't use it. React uses it to distinguish repeated components

Reply:
I think you just need the key here not in the component itself

https://github.com/sarikasoni12/react-ecosystem/pull/1/files#diff-fa71c8cf89d3cd7fd215878cdb33e702691b77649784c5975c0907dd108c03fcR15

Reply:
> I think you just need the key here not in the component itself
> 
> https://github.com/sarikasoni12/react-ecosystem/pull/1/files#diff-fa71c8cf89d3cd7fd215878cdb33e702691b77649784c5975c0907dd108c03fcR15

It required key in component as well otherwise getting console error